buildstockbatch/gcp/gcp.py

```diff
@@ -496,6 +495,41 @@ def get_existing_batch_job(self):
         except exceptions.NotFound:
             return None
 
+    def get_existing_postprocessing_job(self):
+        jobs_client = run_v2.JobsClient()
+        try:
+            job = jobs_client.get_job(name=self.postprocessing_job_name)
+            return job
+        except exceptions.NotFound:
+            return False
+
+    def check_for_existing_jobs(self, pp_only=False):
+        """If there are existing jobs with the same ID as this job, logs them as errors and returns True.
```

Comment:
I wasn't sure if this was going to check for existing Batch/sim jobs, or Run/pp jobs; turns out both. Make this a bit clearer; e.g., "Checks where there are existing Batch/simulations jobs, or Run/post-processing jobs. If so, logs..."